packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotHeader/ChatbotHeader.md

```diff
@@ -10,10 +10,22 @@ id: Chatbot header
 source: react
 # If you use typescript, the name of the interface to display props for
 # These are found through the sourceProps function provided in patternfly-docs.source.js
-propComponents: ['ChatbotHeader']
+propComponents:
+  ['ChatbotHeader', 'ChatbotHeaderMenu', 'ChatbotHeaderActions', 'ChatbotHeaderTitle', 'ChatbotHeaderOptionsDropdown']
```

Comment:
need to add 'ChatbotHeaderSelectorDropdown'

Reply:
Done.